docs/avro.rst

```diff
@@ -127,13 +76,17 @@ field, which is missing in the old data.
     in the Apache Avro project to understand how, for data that was encoded with an older schema, Avro decodes that
     data with a newer, backward-compatible schema.
 
+If the compatibility is set to ``BACKWARD_TRANSITIVE`` (not just ``BACKWARD``), then it means that consumers using the new schema can read data written by producers using all previously registered schemas (not just the latest schema).
```

Comment:
How about this: BACKWARD_TRANSITIVE compatible schema means that version X of the schema can be used to decode data encoded with all versions (X, X-1, ..., 1) of the schema.

Reply:
Good idea. I've adapted this suggestion